app/app.py

```diff
@@ -573,6 +680,10 @@ def main() -> None:
                 with st.chat_message("assistant"):
                     full_response = "Here's what the NHS and Mind each have to say:\n\n"
                     message_placeholder = st.empty()
+                    feedback_placeholder = (
+                        st.empty()
+                    )  # Placeholder for the thumbs up and thumbs down button
```

Comment:
I think this should be fixed right? Is the develop branch not merged into this branch yet?

Reply:
Weird. It is merged. Fixed it again.